src/Segmentation/LevelSets/SegmentWithGeodesicActiveContourLevelSet/Code.py

```diff
@@ -21,10 +21,13 @@
     print(
         "Usage: " + sys.argv[0] + "<InputFileName>  <OutputFileName>"
         " <seedX> <seedY> <InitialDistance> <Sigma> <SigmoidAlpha> "
-        "<SigmoidBeta> <PropagationScaling> <NumberOfIterations>")
+        "<SigmoidBeta> <PropagationScaling> <NumberOfIterations>"
```

Comment:
Is this file's formatting added to this commit as an example of what black does? Or is it unintentional?

Reply:
Yes, this is from black.